DNN Platform/DotNetNuke.Abstractions/Logging/ILogProperties.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// add xml docs.
+    /// </summary>
+    public interface ILogProperties : IList<ILogDetailInfo>, ICollection<ILogDetailInfo>, IEnumerable<ILogDetailInfo>, IList, ICollection, IEnumerable, ICloneable
```

Comment:
@bdukes I updated this to explicitly inherit from `IList<ILogDetailInfo>, ICollection<ILogDetailInfo>, IEnumerable<<ILogDetailInfo>>`. This entire inheritance structure can be simplified but it may have benefit to be explicit

```c#
public interface ILogProperties : IList<ILogDetailInfo>, IList, IClonable
```

Reply:
I think I'd lean towards only requiring `ICollection<ILogDetailInfo>`, so we don't cause issues with other implementations that aren't based on `ArrayList`.  Is the `ICloneable` interface used, or just something that happens to be available on `ArrayList`?

Reply:
`IClonable` is just something that appears on the `ArrayList` interface.

I can update my implementation to just use `ICollection<ILogDetailInfo>` and the existing non-generic interfaces

Reply:
I don't think we need to non-generic interfaces, either.  What benefit/need is there for those?

Reply:
I would support `ICollection<>` ArrayList support is from pre-generics era

Reply:
Fixed in latest code push, resolving thread